testing/compose_files/docker-compose-multiworker.yml

```diff
@@ -219,7 +219,7 @@ services:
       hostname: koku-worker-1-fsfsgr
       image: koku_base
       working_dir: /koku/koku
-      entrypoint: ['watchmedo', 'auto-restart', '--directory=./', '--pattern=*.py', '--recursive', '--', 'celery', '-A', 'koku', 'worker', '-l', 'info', '-Q', 'celery,download,remove_expired,reporting,process,upload,customer_data_sync,delete_archived_data,query_upload']
+      entrypoint: ['watchmedo', 'auto-restart', '--directory=./', '--pattern=*.py', '--recursive', '--', 'celery', '-A', 'koku', 'worker', '-l', 'info', '-Q', 'celery,remove_expired,reporting,process,upload,customer_data_sync,delete_archived_data,query_upload']
```

Comment:
This file no longer needs 10 workers looking at all the queues. We can probably create 1 worker per queue instead

Reply:
Yeah, I will see what else we can trim down from this list.